packages/core/src/checkout/checkout-action-creator.ts

```diff
@@ -20,28 +20,33 @@ export default class CheckoutActionCreator {
         private _formFieldsActionCreator: FormFieldsActionCreator,
     ) {}
 
-    loadCheckout(id: string, options?: RequestOptions): Observable<LoadCheckoutAction> {
-        return concat(
-            of(createAction(CheckoutActionType.LoadCheckoutRequested)),
-            merge(
-                this._configActionCreator.loadConfig({
-                    ...options,
-                    useCache: true,
-                    params: { ...options?.params, checkoutId: id },
-                }),
-                this._formFieldsActionCreator.loadFormFields({ ...options, useCache: true }),
-                defer(() =>
-                    this._checkoutRequestSender.loadCheckout(id, options).then(({ body }) => {
+    loadCheckout(id: string, options?: RequestOptions): ThunkAction<LoadCheckoutAction, InternalCheckoutSelectors> {
+        return (store) => {
```

Comment:
this change will still break core test

also manual test will fail

![image](https://github.com/bigcommerce/checkout-sdk-js/assets/94653751/7629aeeb-43f2-4fff-8d64-9dd32d536c6d)

Reply:
need to find a way to access store config in the function

Reply:
I made another commit (https://github.com/bigcommerce/checkout-sdk-js/pull/2462/commits/448fb6a7015845f149fb7dcf9ba89a88692827a8),

change a bit on observable operations, from 

concat( of(), merge(..., ... , defer()) ) to 
concat(of(), merge(..., ... ), defer()) 

> refer to loadDefaultCheckout.

now manually test on checkout page, it works normally,
but for test there is still one failure, so I'm not sure the change is correct.